README.md

```diff
@@ -50,7 +50,7 @@ for compilation instructions for every supported platform.
 Godot is not only an engine but an ever-growing community of users and engine
 developers. The main community channels are listed [on the homepage](https://godotengine.org/community).
 
-The best way to get in touch with the core engine developers is to join the
+The best way to get in touch with the core engine developers is by joining the
```

Comment:
Again grammar is correct already, "best way is to join" "best way is by joining" is not correct English